### DISCUSSION TITLE
knowledge base grid view for public

### DISCUSSION THREAD

**Discussion by kaganisildak** [GitbookIO/community#471] [Feature Requests]
Gitbook already has grid view in editor mode but there's no option or feature in public website like helpscout. 

In my opinion, both as a customer and as a startup founder, it is very necessary to present this kind of view to the end user when companies with more than one product offer product documentation or knowledge base.

im lost in gitbook collection header view :/

<img width="643" alt="Screenshot 2024-02-02 at 01 28 35" src="https://github.com/GitbookIO/community/assets/12753938/0de288b9-2de5-49a7-b0f2-0b5594182d36">

<img width="1374" alt="Screenshot 2024-02-02 at 01 24 58" src="https://github.com/GitbookIO/community/assets/12753938/e3b33667-6ca1-4a69-9908-27d67a889c1e">

<img width="1160" alt="Screenshot 2024-02-02 at 01 26 07" src="https://github.com/GitbookIO/community/assets/12753938/39ce27b8-4449-47c3-b12b-01050c360b87">

**Comment by taszap** [comment]
Hi @kaganisildak 

Thanks for feedback, noted. 
Have you considered using [Cards](https://docs.gitbook.com/content-editor/blocks/cards)? 
They are similar and could have links and images added in
